Data/3_0/Skills/other.lua

```diff
@@ -1027,7 +1043,6 @@ skills["IcestormUniqueStaff12"] = {
 		"firestorm_drop_ground_ice_duration_ms",
 		"skill_art_variation",
 		"skill_effect_duration_per_100_int",
-		"base_skill_show_average_damage_instead_of_dps",
```

Comment:
You can't remove this line from the other.lua file, everything in here must be modified by only changing the export script.
To get it to show DPS instead of average damage, just add this
`statMap = {
		["base_skill_show_average_damage_instead_of_dps"] = {
		},
	},`
above the base mods section

Reply:
You can look at the Storm Brand gem which does the same thing

Reply:
Thanks I reverted that commit and added hte line you suggested to the other.txt file.